R/summarise.R

```diff
@@ -530,7 +530,8 @@ regional_runtimes <- function(regional_output = NULL,
   if (!is.null(regional_output)) {
     timings <- data.table::data.table(
       region = names(regional_output),
-      time = unlist(purrr::map(regional_output, ~ .$timing))
+      # purrr::map_vec will preserve the difftime class
+      time = unlist(purrr::map_vec(regional_output, ~ .$timing))
```

Comment:
can you update the `@importfrom` tag, too?

Reply:
I've fixed it in [adc114e](https://github.com/epiforecasts/EpiNow2/pull/688/commits/adc114ed1dc27c8bd7f3b7e6a2d3a11fa6a79d3f).